wiring/src/spark_wiring_ble.cpp

```diff
@@ -2192,7 +2188,8 @@ ssize_t BleLocalDevice::getScanResponseData(BleAdvertisingData* scanResponse) co
         return SYSTEM_ERROR_INVALID_ARGUMENT;
     }
     scanResponse->clear();
-    size_t len = CHECK(hal_ble_gap_get_scan_response_data(scanResponse->data(), BLE_MAX_ADV_DATA_LEN, nullptr));
+    CHECK_TRUE(scanResponse->resize(BLE_MAX_ADV_DATA_LEN), SYSTEM_ERROR_NO_MEMORY);
```

Comment:
Not sure but we should probably need to resize according to the PHY as what we do to `getAdvertisingData()`?

Reply:
Scan response is only available when using 1MBPS PHY, so I think that keeping the maximum resize to `BLE_MAX_ADV_DATA_LEN` which is 31 bytes is ok.